CHANGELOG.md

```diff
@@ -1,119 +1,19 @@
-# 17.0.0
```

Comment:
This seems to delete the 17.0.0 section in CHANGELOG.md. Maybe this needs a merge conflict resolution?